plugins/catalog/src/index.ts

```diff
@@ -18,6 +18,7 @@ export * from './components/AboutCard';
 export { CatalogLayout } from './components/CatalogPage';
 export { CatalogResultListItem } from './components/CatalogResultListItem';
 export { CatalogTable } from './components/CatalogTable';
+export type { EntityRow } from './components/CatalogTable';
```

Comment:
I'd vote for renaming this to `CatalogTableRow` as part of exporting it, to group things up a bit more